lib/setup/setupPacketManager.js

```diff
@@ -67,7 +67,7 @@ class PacketManager {
             if (e.stderr === '') {
                 return true;
             }
-            console.error(e);
+            console.error(e.stderr || e.stdout || e);
```

Comment:
do not log uglys constructs like https://github.com/AlCalzone/ioBroker.ble/pull/525/checks?check_run_id=1803728242#step:6:163